blocks-config/forms/class-uagb-forms.php

```diff
@@ -73,12 +73,32 @@ public function process_forms() {
 				'recaptcha_secret_key_v3' => \UAGB_Admin_Helper::get_admin_settings_option( 'uag_recaptcha_secret_key_v3', '' ),
 			);
 
-			if ( 'v2' === $_POST['captcha_version'] ) {
+			if ( ! isset( $_POST['post_id'] ) || empty( $_POST['post_id'] ) || ! isset( $_POST['block_id'] ) || empty( $_POST['block_id'] ) ) {
+				wp_send_json_error( 400 );
+			}
+
+			$post_content = get_post_field( 'post_content', $_POST['post_id'] );
```

Comment:
Sanitize data before use - `$_POST['post_id']`
```suggestion
			$post_content = get_post_field( 'post_content', $_POST['post_id'] );
```